fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -194,6 +194,20 @@ def reset_all
         all.each(&:reset)
       end
 
+      def copy_logarchive(device, dest)
+        sim_resource_dir = FastlaneCore::CommandExecutor.execute(command: "xcrun simctl getenv #{device.udid} SIMULATOR_SHARED_RESOURCES_DIRECTORY 2>/dev/null", print_all: false, print_command: true)
+        logarchive_src = File.join(sim_resource_dir, "system_logs.logarchive")
+        FileUtils.rm_rf(logarchive_src) if File.exist?(logarchive_src)
```

Comment:
I think that @lacostej was suggesting that it isn't necessary to check if the file exists as the `rm_rf` will delete it if it does and not complain if it does not.